newa/cli.py

```diff
@@ -41,35 +41,55 @@
     )
 
 JIRA_NONE_ID = '_NO_ISSUE'
+STATEDIR_PARENT_DIR = '/var/tmp/newa'
```

Comment:
It should be set to a `Path` instance right here, to prevent any path-like string sneaking in.

Reply:
fixed